clientv3/lease.go

```diff
@@ -120,6 +120,14 @@ func NewLease(c *Client) Lease {
 	l.stopCtx, l.stopCancel = context.WithCancel(context.Background())
 	go l.recvKeepAliveLoop()
 	go l.deadlineLoop()
+
+	go func() {
```

Comment:
I think this should really be a `l.stopCancel()` in `recvKeepAliveLoop()`'s `defer`